src/test/java/seedu/multitasky/logic/commands/RestoreCommandTest.java

```diff
@@ -0,0 +1,119 @@
+package seedu.multitasky.logic.commands;
+
+import static org.junit.Assert.assertTrue;
+import static seedu.multitasky.testutil.SampleEntries.INDEX_FIRST_ENTRY;
+import static seedu.multitasky.testutil.SampleEntries.INDEX_SECOND_ENTRY;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+
+import org.junit.Test;
+
+import seedu.multitasky.commons.core.Messages;
+import seedu.multitasky.commons.core.index.Index;
+import seedu.multitasky.logic.CommandHistory;
+import seedu.multitasky.logic.parser.Prefix;
+import seedu.multitasky.model.Model;
+import seedu.multitasky.model.ModelManager;
+import seedu.multitasky.model.UserPrefs;
+import seedu.multitasky.model.entry.Entry;
+import seedu.multitasky.model.entry.ReadOnlyEntry;
+import seedu.multitasky.testutil.SampleEntries;
+
+//TODO implement restore by find tests and separate out restore by index / abstract class restore tests from here.
+
+//@@author A0126623L reused
```

Comment:
for reused code put it as `A0126623L-reused`. it ignores everything after whitespace.